server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -128,22 +129,22 @@
                 if (fieldName.equals("type")) {
                     continue;
                 }
-                if (Fields.ANALYZER.match(fieldName)) {
+                if (Fields.ANALYZER.match(fieldName, LoggingDeprecationHandler.INSTANCE)) {
```

Comment:
I suppose this is the right thing to do but, man, I don't really like it. Oh well. +1

Reply:
Yeah I'm not a huge fan of it either, but not many options